api/v1/ytsaurus_types.go

```diff
@@ -194,7 +194,8 @@ type BundleBootstrapSpec struct {
 	SnapshotPrimaryMedium  *string `json:"snapshotMedium,omitempty"`
 	ChangelogPrimaryMedium *string `json:"changelogMedium,omitempty"`
 	//+kubebuilder:default:=1
-	TabletCellCount int `json:"tabletCellCount,omitempty"`
+	TabletCellCount     int     `json:"tabletCellCount,omitempty"`
+	TabletNodeTagFilter *string `json:"tabletNodeTagFilter,omitempty"`
```

Comment:
Let's drop `Tablet` so that the name of the variable corresponds to the name of the attribute being set.
(there is no point in filtering any other nodes beside tablet nodes for tablet cell bundles, so it is irrelevant)